### PR TITLE
Invoke-VCFCommand fix + Get-VCFVersion fix

### PR DESCRIPTION
**Pull request by giulianoberteo**
Summary of the commits included in this PR:

- Get-VCFVersion (FIX): had Select statement which was causing substring method to fail + wrong substring index was used
- Resolve-PSModule (NEW): required by Invoke-VCFCommand
- Invoke-VCFCommand (ENHANCED): new SoS options + bug fix
- Invoke-VCFCommand markdown documentation (NEW) + typos fixed

### CODE REVIEW

File: powerVCF.psm1 [hunk 1]
```diff
@@ -2558,28 +2558,24 @@ Export-ModuleMember -Function Get-VCFvRLI
...
+            If passwords are not passed as parameters it will prompt for them.
     
         .EXAMPLE
-        PS C:\> Connect-VCFCommand -fqdn sfo01vcf01.sfo.rainpole.local -vcfpassword VMware1! -rootPassword VMware1! -sosOption general-health 
+        PS C:\> Invoke-VCFCommand -vcfpassword VMware1! -rootPassword VMware1! -sosOption general-health 
         This example will execute and display the output of "/opt/vmware/sddc-support/sos --general-health" command
     
         .EXAMPLE
-        PS C:\> Connect-VCFCommand -fqdn sfo01vcf01.sfo.rainpole.local -sosOption general-health 
+        PS C:\> Connect-VCFCommand -sosOption general-health
```
Comment (PowerVCF) [inline powerVCF.psm1]: Should this be Invoke instead of Connect?

File: powerVCF.psm1 [hunk 2]
```diff
@@ -2558,28 +2558,24 @@ Export-ModuleMember -Function Get-VCFvRLI
...
         .SYNOPSIS
-        Connects to the specified SDDC Manager using SSH and invoke SSH commands (SOS)
+        Connects to the specified SDDC Manager using SSH and invoke SSH commands (SoS)
     
         .DESCRIPTION
-        The Invoke-VCFCommand cmdlet connects to the specified SDDC Manager via SSH using vcf user and 
-        subsequently execute elevated SOS commands using the root account. This is why both vcf and root password are
-        mandatory parameters. If passwords are not passed as parameters it will prompt for them.
+        The Invoke-VCFCommand cmdlet connects to the specified SDDC Manager via SSH using vcf user and subsequently 
+        execute elevated SOS commands using the root account. Bboth vcf and root password are mandatory parameters.
```
Comment (PowerVCF) [inline powerVCF.psm1]: Typo "Bboth"

File: powerVCF.psm1 (hunk 1 above)
Comment (giulianoberteo) [reply powerVCF.psm1]: that's indeed already been fixed in commit "typos fixed" if you see :-)

Comment (PowerVCF) [reply powerVCF.psm1]: Ok cool. I thought it would show in the compare if it was part of the PR.

File: powerVCF.psm1 (hunk 2 above)
Comment (giulianoberteo) [reply powerVCF.psm1]: fixed just now in commit "Invoke-VCFCommand typos fixed on synopsis/description"

**Pull request merged**